app/controllers/sufia/admin/admin_sets_controller.rb

```diff
@@ -1,7 +1,42 @@
 module Sufia
   class Admin::AdminSetsController < ApplicationController
+    include CurationConcerns::CollectionsControllerBehavior
     load_and_authorize_resource
     layout 'admin'
+    self.presenter_class = Sufia::AdminSetPresenter
+
+    def show
+      add_breadcrumb t(:'sufia.controls.home'), root_path
+      add_breadcrumb t(:'sufia.toolbar.admin.menu'), sufia.admin_path
+      add_breadcrumb t(:'sufia.admin.sidebar.admin_sets'), sufia.admin_admin_sets_path
+      add_breadcrumb 'View Set', request.path
+      super
+    end
+
+    # Override the default prefixes so that we use the collection partals.
+    def _prefixes
```

Comment:
This is not a public API. We discussed this on Slack the other week and @cbeer :speech_balloon: 
> we override `#_prefix` because `.local_prefixes` will prepend the path, and we really want to append it.